.github/workflows/ci-tests.yml

```diff
@@ -203,7 +211,7 @@ jobs:
       # Do not set explicitly temp dir for dbt as causes problems
       # None of these test need temp dir set
       run: |
-        python -m tox -e dbt018-winpy -- plugins/sqlfluff-templater-dbt -m "not dbt_connection_failure and not needs_dbt_connection"
+        python -m tox -e dbt018-winpy -- plugins/sqlfluff-templater-dbt
```

Comment:
we don't need to exclude these anymore (removed these unused marks in the test code as well)